src/map/packet_system.cpp

```diff
@@ -735,6 +735,12 @@ void SmallPacket0x01A(map_session_data_t* const PSession, CCharEntity* const PCh
     // uint32 ID = data.ref<uint32>(0x04);
     uint16 TargID = data.ref<uint16>(0x08);
     uint8  action = data.ref<uint8>(0x0A);
+    position_t actionOffset =
+    {
+        data.ref<float>(0x10),
```

Comment:
General question, are we guaranteed that float on all architectures will always be 4 bytes?  If not, we might want to pull this in as something else and cast.

Reply:
Yeah, that's a pretty safe assumption 👍

Reply:
I think we can also rely on the endianness being correct, as we pull floats out of packets in other places